spec/lib/trade_tariff_admin_spec.rb

```diff
@@ -0,0 +1,90 @@
+require "rails_helper"
```

Comment:
You actually don't need these requires https://github.com/trade-tariff/trade-tariff-admin/blob/HMRC-1529/passwordless-authentication/.rspec#L1